### ISSUE TITLE
About featured.json

### ISSUE THREAD

**Issue by apple502j** [LLK/scratch-www#1949]
/src/views/splash/featured.json
Is it needed? Splash now uses API, so I think we can delete it.

**Comment by thisandagain** [comment]
Good catch. This file was for testing early on in the development process and can be safely removed.
